Feature/HMI Policy Table Update using vehicle modem (#185)
smartdevicelink/sdl_hmi_integration_guidelines: docs/SDL/UpdateSDL/index.md

1) UP_TO_DATE – Policy Table is now up to date, but anyway SDL starts a new policy update cycle because of user request (e.i. UpdateSDL).   
2) UPDATING – Policy Update process is now in already progress, anyway when currently active update process will be finished, SDL must start PT Update from the beginning.   
3) UPDATE_NEEDED – Policy Table is not up to date, update process must be started by SDL.   

> PoliciesManager must initiate the PT Update sequence (that is, PT Exchange) upon User's request delivered to SDL via SDL.UpdateSDL() from HMI and provide a response on a request with current PTU status to HMI.

Reviewer: ```suggestion

PoliciesManager must initiate the PT Update sequence upon user's request, which is delivered to SDL via `SDL.UpdateSDL()` from the HMI,  and provide a response with current the PTU status to the HMI.
```
